docs/index.md

```diff
@@ -1,4 +1,51 @@
 # Ansible PROJECTNAME Documentation
 
-Provide a brief summary of the Ansible project and why someone would want to use it.
-The summary should be two or three sentences long.
\ No newline at end of file
+> Provide a mission statement or a brief summary of the Ansible project and why someone would want to use it. It should be two or three sentences long, for example:
+
+At the `PROJECTNAME`, our mission is to produce and maintain simple, flexible,
+and powerful open-source software tailored to `PROJECT_PURPOSE`.
+
+We welcome members from all skill levels to participate actively in our open, inclusive, and vibrant community.
+Whether you are an expert or just beginning your journey with Ansible and `PROJECT_NAME`,
+you are encouraged to contribute, share insights, and collaborate with fellow enthusiasts!
+
+> Please have at least the following sections filled up with relevant information. Depending of your project complexity, you can add additional sections, for example, a separate Getting started guide and so on.
+
+- [License](LICENSE link here)
+- [Community guide](community_guide.md)
+  - [Code of Conduct](community_guide.md#code-of-conduct)
+  - [Communication](community_guide.md#communication)
+    - [Asynchronous channels](community_guide.md#asynchronous-channels)
+    - [Real-time channels](community_guide.md#real-time-channels)
+
+- [User guide](user_guide.md)
+  - [Communication](user_guide.md#communication)
+  - [Getting started](user_guide.md#getting-started)
+  - [Installation guide](user_guide.md#installation)
+  - [Next section](user_guide.md#next-section)
+
+- [Contributor guide](contributor_guide.md)
+  - [Communication](contributor_guide.md#communication)
+  - [Project place in Ansible ecosystem](contributor_guide.md#project-place-in-ansible-ecosystem)
+  - [Non-code contributions](contributor_guide.md#non-code-contributions)
+    - [Ask and answer questions](contributor_guide.md#ask-and-answer-questions)
+    - [Contributing to documentation](contributor_guide.md#contributing-to-documentation)
+    - [How to open an issue](contributor_guide.md#how-to-open-an-issue)
+  - [Code contributions](contributor_guide.md#code-contributions)
+    - [Project architecture](contributor_guide.md#project-architecture)
+    - [Getting started](contributor_guide.md#getting-started)
+    - [Testing](contributor_guide.md#testing)
+    - [Reviewing open issues and pull requests](contributor_guide.md#reviewing-open-issues-and-pull-requests)
+
+- [Maintainer guide](maintainer_guide.md)
+  - [Communication](maintainer_guide.md#communication)
+  - [How to become a maintainer](maintainer_guide.md#how-to-become-a-maintainer)
+  - [Maintainer responsibilities](maintainer_guide.md#maintainer-responsibilities)
```

Comment:
Should we list what a maintainer does before how to become one?

Reply:
wasn't sure, let's move it then